AUTHORS

```diff
@@ -11,6 +11,7 @@ Patches and Suggestions
 
 - Adam Zapletal
 - Ali Afshar
+- Andrii Soldatenko
```

Comment:
Not sure if that makes lots of sense for such a small change (even though such changes are very welcome of course!)

There are commits from 368 people in this repo, but way less entries in this file (adding every contributor would quickly make the file pretty chaotic).

IMO it'd be better to only add yourself to this file when you contribute e.g. a major fix, new feature, etc